test/LaunchDarkly.EventSource.Tests/LaunchDarkly.EventSource.Tests.csproj

```diff
@@ -1,13 +1,19 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.1;net461</TargetFrameworks>
+    <!-- The TESTFRAMEWORK variable allows us to override the target frameworks with a
+         single framework that we are testing; this allows us to test with older SDK
+         versions that would error out if they saw any newer target frameworks listed
+         here, even if we weren't running those. -->
```

Comment:
I've done this kind of thing in most of our .NET libraries, ever since we started running CI tests for newer frameworks like .NET 5.0. The issue is that when we run the CI job for, say, .NET Core 2.1, we want to actually use the .NET Core 2.1 SDK— but that SDK will complain if it even sees `net5.0` listed here in the project file.